components/fxa-client/ios/base.xcconfig

```diff
@@ -3,4 +3,5 @@
 INFOPLIST_FILE = FxAClient/Info.plist
 LIBRARY_SEARCH_PATHS = "../../../target/universal/$(buildvariant)"
 FRAMEWORK_SEARCH_PATHS = "../../../Carthage/Build/iOS"
-SWIFT_OBJC_BRIDGING_HEADER = FxAClient/FxAClient.h
+CLANG_ENABLE_MODULES = YES
```

Comment:
`/xcconfig/common.xcconfig"` would be the best place

Reply:
ahhh shit... didn't know we had an overarching common config... but yeah, they'd be better in there